CHANGELOG.rst

```diff
@@ -16,6 +16,10 @@ User-Facing Changes
 * 📑 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
   accessed projects, and own datasets
   (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
+* ⚙️ **Renku CLI**: Connecting to Renkulab sessions via SSH
+  (`#3318 <https://github.com/SwissDataScienceCenter/renku-python/pull/3318>`_).
+* 📃 **Renku CLI**: Support for Azure Blob Storage backed datasets.
+  (`#3257 <https://github.com/SwissDataScienceCenter/renku-python/pull/3257>`_).
```

Comment:
This bullet point is already coming via the [Renku Notebooks PR](https://github.com/SwissDataScienceCenter/renku/pull/2967/files) (and also it's in the "Internal Changes" section, since this feature is experimental and not being rolled out to users broadly).
```suggestion
```